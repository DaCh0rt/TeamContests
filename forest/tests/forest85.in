818 472
143 6149
-8749 133
9330 -6423
5905 7469
-1797 -9978
-1650 5120
5699 -1897
2452 -4975
7241 8751
-4327 6943
-1944 9748
4509 -4820
6 -7491
6193 -9468
8674 9519
-7884 -5345
3120 -4363
-3251 6354
-8910 4596
-4675 2641
5813 -433
5512 7738
2603 -2705
-2530 -6917
6123 8804
-3647 9697
-9964 -9514
906 9728
2797 -3858
-8825 4075
-4941 2954
571 -8334
6189 -8981
9907 -7201
-8583 9576
-492 9038
-3071 -7800
-703 -2195
-1440 9838
-6223 2382
-8378 -3836
1734 5280
-3789 -1385
-1 -8021
3723 3935
-4355 5106
5148 4151
7460 -5903
-2113 3363
2864 7468
2899 3456
-1536 -3625
4915 1423
1398 1178
-7350 3072
2994 -1640
1665 -661
-5584 7100
842 1262
-9158 -7087
-6120 -136
-4094 9544
-6288 -6272
-928 6545
-9711 -4258
7451 -819
-8488 6570
8850 -5999
-2976 -5034
1319 -8102
-7673 1523
-1610 5101
7272 -7131
-9576 -1488
3958 3370
-4635 -1785
7401 5829
-820 7637
-2869 2148
1985 2058
-3769 7820
6398 -4716
8714 -3072
-136 -487
-4074 4326
-8037 -6297
6312 -4154
6392 -2765
-5688 -7372
-5838 -7194
-3687 5800
-9303 -979
-467 -4872
7215 -6665
-6496 -6385
3314 -9386
-1905 1102
5820 -1862
-1836 7192
-5607 7293
-6846 -2695
464 3144
6596 -8111
5159 4064
-5728 2903
-886 1008
6384 8621
-3741 -5273
3537 -4091
3424 -3058
-4895 7068
-7738 905
4282 -7500
-2386 -7561
-9150 9702
-5110 -3912
-2486 -3436
3610 736
1456 -6552
2408 -6475
3720 -1307
-5279 8549
-550 -5094
7375 -1478
2142 -3693
4003 325
-88 -8121
496 2234
-7242 6758
9701 5622
-4194 -3065
-3375 7297
8441 2562
-1302 -3277
6052 -8352
570 -4845
-9257 -3170
-8832 -383
8977 7875
5759 -9681
-1031 -3756
7838 9040
-3593 8659
7424 -616
-6012 -7391
6802 -4850
4468 1463
1241 -5172
-2181 4085
4301 3346
5594 -4062
-6543 -4243
-8378 1878
2063 -9723
-7461 -9530
8350 -731
3564 -9039
-3592 -1121
-3359 2703
-1231 -4040
5964 -8712
-9894 7107
-9133 -8792
8444 -2619
-6709 -8138
-6052 -9308
1127 43
714 2752
-9668 4510
2040 -5627
5372 -5192
-5259 9578
1951 4210
8656 -5156
5816 919
-4797 8693
8370 -1977
1941 4758
6694 -4709
8479 6488
-3410 7387
8418 2534
-3623 2371
515 4719
-4636 -4483
-649 213
513 -8284
-8685 7993
1694 -9092
-7593 -2505
-3590 8933
-7323 -57
-3800 -4894
5666 -8934
-280 -5678
5570 2160
-861 9081
-5582 1012
7234 -431
-7365 3517
6364 -7934
-9216 -6749
-4596 8601
-3129 -1176
5661 -2490
-3702 -7084
-225 8704
-5327 1086
7958 -4507
660 -6107
1937 2997
647 3523
3510 3057
8448 9901
3266 3850
5595 4813
153 8217
-841 -171
7151 7161
4316 9331
-2575 8294
-3878 8404
-80 -5525
832 -7749
-7632 3377
-9388 9270
-882 9835
-2833 -3989
-3385 5529
4255 -2001
309 7915
-8319 6335
-4603 -2027
313 7259
-3579 -1535
5504 -2815
3356 -3246
-9026 7141
-365 7205
-6154 -573
3636 562
-4744 -5694
5544 1041
-1024 -4612
1430 3043
-3337 -2125
7174 -1031
-7679 -2160
-614 -6121
-9154 -3123
-1737 -6548
2689 4435
-7624 6267
3041 -5271
1991 -401
160 -9968
9020 3422
8504 2283
-2250 9650
-3165 -1765
1973 9652
2602 -5546
2260 7704
-657 -8958
1150 7395
-9324 4401
-8806 -6014
-4601 -9330
2738 -2835
472 -9046
-1916 -8342
-8956 4446
546 -9796
3865 -9778
-1679 -4717
-6831 -6515
7227 -9121
-2627 -9247
-5038 -4313
-2983 -7033
7996 -7404
5778 3218
-9387 -6883
-7348 9360
-6988 -1653
6831 7728
6271 -8290
7031 8719
-147 -2793
9862 -9071
-7416 1621
4736 3021
2610 -9264
-6565 9091
-706 6302
-3942 453
4972 -2654
-4831 -1491
4310 -7288
-8352 -4681
-9422 -5099
-8762 2986
-8242 5009
3326 8357
-6463 9816
-2681 6478
-1526 7767
-6674 -6363
-937 2995
9839 -4951
7901 -6043
-1935 -629
2925 8951
-8593 -7460
3093 -1803
-3557 -7834
8433 -7876
9203 9809
-2107 526
-379 6205
-3287 4740
9353 946
734 5360
7467 -8912
-7563 -4728
-4836 872
3773 -5129
-8396 -3780
-7459 6962
5514 9185
4657 -410
-9643 -708
3369 7219
7073 -3360
13 6601
-9266 -9874
-1093 -2430
2097 9665
6935 3631
-3469 6802
-1406 -7674
-2105 1294
-8081 3987
-9226 -9963
-257 -6405
-9960 -5322
7374 1598
-9005 8062
7230 6497
-9505 -1273
9974 3508
2007 -7505
-2070 -4729
1582 9421
-7663 -821
-1438 5699
6753 3223
-6369 -4151
9290 -6645
-5570 -4619
-2854 9184
-7487 9634
1282 -5561
-3089 -1492
2850 9425
3832 7605
6698 4330
3880 1780
3986 6220
1019 -1416
2801 8662
6770 -3096
-7119 5426
8203 -4673
-8543 -50
6526 5506
6327 7445
-3719 -5126
3272 -4431
-3044 -8163
6343 9029
-9286 3660
5539 3667
-5373 -3195
-3919 -1492
996 4824
-216 6052
-7548 484
9714 6682
7700 3291
-5667 8461
2600 3813
-3983 642
-8817 -1906
2085 -2610
577 3396
2877 5008
3855 -9696
-4844 -722
474 5400
-4481 -3457
-5341 -295
-4204 -5876
-6610 9812
-335 2013
8448 -1828
7867 7798
4992 3121
-3695 -6182
-9281 -6664
-1198 -2041
-2568 -7074
-293 -3570
9892 -1243
4971 -220
416 -721
8888 -8046
2634 -7131
-7868 5368
8907 -3325
-7645 8538
-1101 7775
3662 5977
5719 -3959
-9079 9349
51 5302
8121 7587
-6887 -8148
-2552 6695
-595 8491
1144 508
-9662 -8175
-8816 -4004
9081 6959
8963 2162
-6078 -6986
-4360 -5658
6154 4311
-1651 3712
-4917 8266
-855 -1468
1789 -8958
4195 2043
-3802 -3138
-7580 -1187
7833 -6118
3073 -3656
-8363 273
5955 7853
-9406 -8002
3497 3146
7596 -2073
-2380 -6046
3112 -8019
6031 -346
3695 -4914
-5640 6472
2097 3846
-142 6686
-7128 -1118
5237 -8627
3101 2418
-3721 3177
9967 -7356
-3571 3006
-7503 3245
-5567 3833
-2119 -8306
4183 7888
8468 1354
1268 -8618
-8714 -4950
-7814 -3962
-5611 -9526
5999 792
8991 -6281
3909 3072
4609 7188
4130 7824
-5891 -4236
3449 576
-7065 -9676
5673 262
-2182 246
-4780 -4497
-7017 957
1473 2270
8243 7521
-4413 -5966
-4290 -5335
4568 4628
2075 1818
6973 -6994
-5544 5738
-7215 -7501
-6486 -5954
-7114 9982
-3270 914
535 4694
5600 7398
-9307 764
-466 7153
-6394 -2153
4277 -6189
-8561 -1763
5270 -9398
-3124 7497
2437 1238
7800 2854
-5444 -8327
9685 9269
9617 8239
-8746 -2180
-4392 -9141
1138 -1275
-417 -6727
8736 -4732
9429 2755
-515 2180
-3049 -7504
-5720 -7481
-7510 7230
3710 -1762
-1992 5834
505 329
-4990 8808
-5917 9282
-6701 -47
1315 9519
3427 9335
1734 -75
-2220 -8807
-5910 9677
-7762 5722
986 -330
-7535 -5601
-9960 -2641
7049 -7318
-2649 -1305
7070 6248
5483 -9264
8382 -2997
3224 9039
8008 6326
8926 9614
-8555 -1825
-1475 -1516
1202 -1280
6039 -8033
5673 -1972
8584 9538
4314 2937
-8686 -1543
-6160 -2962
-9489 3620
1523 -482
-4 -4187
8459 -7746
1670 3927
-7715 9988
9206 -7935
1063 2283
-3458 -8317
-3232 -3862
1410 -1183
2368 -7426
1158 9834
7121 -1511
-2278 -6541
4907 -6365
-5350 8492
9928 -4869
8270 4952
6952 -6456
3954 8878
1208 -7414
-4538 -7029
-734 -8046
-5410 -6477
-4216 6545
1135 1811
7757 5382
-9663 8241
-9857 6857
-4185 -5156
3388 9108
244 3927
3435 -6136
-4952 -7704
1710 3667
-3153 -2866
-2348 6877
-2839 9899
-5905 -8916
5452 6495
1210 -2601
8763 4671
-4434 -8868
-11 3419
8334 -651
-5714 5567
-4561 -4624
1168 -1651
743 4757
3169 7363
7766 -2791
-4037 1451
-3108 -9104
-6401 -1801
-3212 -2863
-7402 -8550
5139 -8908
5448 -3080
2517 -3955
5294 819
5047 556
-3187 -8315
-2399 1969
-8113 8351
2025 -1812
973 -6106
-6653 8406
-7179 5815
-2159 -2348
1001 -8456
3283 8738
-6034 -51
-8116 7403
7011 -9676
-7998 -8355
3622 -7515
-7160 -1893
-3022 -5235
4619 -2183
-9959 4032
7665 6485
-8610 1612
2610 8481
6460 -591
-6492 807
-7778 906
7346 -836
-6657 -9452
-1321 -928
5254 -3152
4655 6328
-7554 -6895
6903 9542
-9922 6704
6932 -2810
2603 -4236
-3743 -123
2898 -5553
4746 -4533
-908 5822
-4845 6475
5629 1657
-3571 -5728
-2575 1299
-7756 6740
-841 2722
3882 -6936
-9531 2579
2497 -699
-2810 2946
5844 -3223
-7813 7056
6194 2502
5887 -8783
-2527 5738
-6711 8283
309 -5199
-8292 -5594
-5479 -2650
6019 -786
-8855 4038
4778 1890
4031 -5480
-3714 -4085
8618 4344
2730 -4605
-646 8226
938 6199
9687 6781
1972 4187
2007 -72
-789 616
4123 -4353
-6252 -7419
2865 5306
3356 -306
5851 1229
-8326 -6684
-3288 9406
-1846 762
2399 2245
941 -845
9725 -6635
8337 491
24 7510
6678 -3404
8029 -7061
4147 -8243
8799 -8049
-3966 -2381
-1941 -1779
-9658 7213
6779 8970
658 -3796
5733 1289
-9741 922
-1411 -4348
-1827 -8575
-2554 -1385
3646 6799
6149 5402
-5481 1140
8129 -5293
9023 -3067
2267 6676
6465 -625
2971 3677
1727 4232
3332 1241
-937 8118
-7282 2379
-1757 -742
8239 9002
-2675 3432
6552 1417
-2214 7169
-9655 4228
1720 -6442
-1438 4582
-553 -8704
1879 4225
2875 -1012
7917 -5543
6177 2943
-4150 4682
8861 7677
-6787 -194
-4856 -9262
9415 -4729
-3574 -6707
-4260 -1321
-9001 6453
4855 -9618
-1920 3695
2806 9267
-6030 1185
4042 -7782
-1483 -2705
-7361 5749
9333 -3030
-9886 -6789
-8101 -8141
6258 2884
-6819 -2991
262 -3711
-7327 1250
-8504 -3708
2188 -8373
8875 -9968
-8117 8904
295 1899
-5716 -9858
-8200 2341
8909 4085
-1113 2041
-9550 2287
8338 -9586
1353 6031
175 2754
-9222 -9751
9535 -1668
-2642 -1419
1003 -8165
-4698 -9366
-4243 -5439
-3373 -9935
8588 6361
8523 -4143
-3008 -2822
-5413 -9970
4923 8016
-5828 8413
-1220 -2723
176 5134
1728 6477
9498 9785
5879 -5131
-2830 -1745
2864 1113
-8293 504
-181 4952
-1519 7081
-5237 -3640
-8449 -9102
-3920 7376
-8798 3748
3789 7840
8513 -3574
929 790
-776 -4855
412 -165
-7276 582
-9953 6461
8519 542
5379 -2198
917 -5455
3783 5731
-6683 5633
8886 8341
-9115 7297
-1863 -6962
-3948 -2468
614 -1044
