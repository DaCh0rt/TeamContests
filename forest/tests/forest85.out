80185.71 430788762.48
